renderer/plugin.js

```diff
@@ -138,6 +138,47 @@ org.ekstep.contentrenderer.questionUnitPlugin = Plugin.extend({
    */
   getQuestionState: function() {
     return this._question.state;
+  },
+  /**
+   * provide media url to asset, runs inside the context of [mtf, fib, mcq context]
+   * @memberof org.ekstep.questionunit
+   * @param {String} url from question set.
+   * @returns {String} url.
+   */
+  getAssetUrl:function(url){
+    if(isbrowserpreview){// eslint-disable-line no-undef
+      return url;
+    }
+    else{
+      return 'file:///' + EkstepRendererAPI.getBaseURL()+ url;
+    }
+  },
+  /**
+   * play audio once at a time, runs inside the context of [mtf, fib, mcq context]
+   * @memberof org.ekstep.questionunit
+   * @param {String} audio from question set.
+   */
+  playAudio: function(audio){
+    audio = this.getAssetUrl(audio);
+    if (this._lastAudio && (this._lastAudio != audio)) { // eslint-disable-line no-undef
```

Comment:
We should have proper interface methods for the Audio functionality. How this base plugin knows what is the _lastAudio played?

If there is a background Audio(auto play) for title, How this is handling those?

Reply:
Currently I am discussing it with @ramj-funtoot, Will update on this once done